enterprise/backup/src/main/java/org/neo4j/backup/impl/ParametrisedOutsideWorld.java

```diff
@@ -32,7 +32,7 @@
 /**
  * An outside world where you can pick and choose which input/output are dummies.
  */
-class ParametrisedOutsideWorld implements OutsideWorld
+public class ParametrisedOutsideWorld implements OutsideWorld
```

Comment:
Should this be in test sources?

Reply:
It should definitely have a `test` scope. It is only used in backup module tests at the moment.

Reply:
Resovled -> moved to `command-line` module `src/main` and kept `public`